### PR TITLE
Fixed :before css content shown in Text

### PR DESCRIPTION
**Pull request by phasze**
Fixed before/after css content shown in Text

### CODE REVIEW

**Comment by phasze** [comment]
Trying to get this goog.style to work but it never returns anything.  Any ideas jleyba?

**Comment by andreastt** [comment]
I'm not saying it's a blocker, but to get this right we also have to iterate the child nodes of `elem` in case they have any before/after rules attached.  That would make the algorithm recursive, and so this should be extracted in a subroutine.

**Comment by jleyba** [comment]
Sorry, told you to do the wrong thing with goog.style. Doesn't look like closure supports pseudo elements (should fix that). There's still the issue of `window` not guaranteed to be the element's window. So what we need is:

    if (!bot.userAgent.IE_DOC_PRE9) {
      var content = goog.dom.getOwnerDocument(elem).defaultView.
          getComputedStyle(elem, ':before').content;
    }

File: javascript/atoms/dom.js [hunk 1]
```diff
@@ -1136,10 +1136,28 @@ bot.dom.appendVisibleTextLinesFromElement_ = function(elem, lines) {
...
+      afterContentCSS = goog.style.getComputedStyle(elem, ':after').content;
+      if(!bot.userAgent.IE_DOC_PRE9 && beforeContentCSS.length > 0 && beforeContentCSS!='none'){
+      //trim first and last characters because content puts quotes (single in some browsers and double in others) inside the string.
+        beforeContentCSS=beforeContentCSS.substr(1,beforeContentCSS.length-2);
+        if(lines.length==0)
+          lines.push(beforeContentCSS);
+        else
+          lines[lines.length-1]+=beforeContentCSS;
+      }
+      if(afterContentCSS.length > 0 && afterContentCSS!='none'){
```
Comment (jleyba) [inline javascript/atoms/dom.js]: You should not append the `:after` content until after elem.childNodes have been processed.

**Comment by phasze** [comment]
Cool, that worked like a charm.

**Comment by jleyba** [comment]
This needs to be updated to handle the `:after` pseudo-class before it can be merged - it shouldn't require much more work and will provide a complete addition to isShown computations.

**Comment by andreastt** [comment]
@phasze, do you have opportunity to fix up the last remaining code changes needed to merge this?

**Comment by AutomatedTester** [comment]
Closing as review comments were never addressed

**Pull request closed without merging**